### PR TITLE
Production OTA: Add link for offline keys (requirement) and reason for motivation to do static deltas

### PR DESCRIPTION
**Pull request by camilamacedo86**
Signed-off-by: Camila Macedo <camila.macedo@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 464ddee12bb5676b00e87a0ee3077900575668b2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2386/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 34969c0a45d8801230a19aad440b7d24089fa5ed are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2387/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 7bd0383c68ef940329aaf90125fa82b7a8be6cb6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2388/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for c7d77275ee0a617a284140ee70ca9096cd8be61a are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2389/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 5b85a90f98f2523738688f96a6b6c65107182b50 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2390/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 6c433e17b0d1099d670eeac4cf224a9673401c0e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2391/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 035fd3483a5915a46edbe70c7e8a8a5b7648e7bf are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2392/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for f58974e9e5a78602e0db156ca2966e3cf8227160 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2393/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 499ad8fa3717a239f43638829236cd139ed6aa0d are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2394/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for f2ce3da6432db645717080cc54be6c38b56e5aab are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2395/docs/artifacts/html/index.html

File: source/reference-manual/ota/production-targets.rst [hunk 1]
```diff
@@ -45,9 +51,24 @@ To start the production release process, a user would create a new wave using th
 This creates a new TUF targets role version for production devices which listen to OTA updates for the ``production`` tag.
 That TUF targets role only includes a single Target from CI build (in above example, that target version is 42).
 
-.. note::
+.. important::
 
    We recommend that a user generates :ref:`OSTree static deltas<ref-static-deltas>` before rolling out waves to devices.
+   Static Deltas will optimize your OTA update download.
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/ota/production-targets.rst:57]: As I read I just checked "we recommend X"
But it is missing why. I would like to have this info before deciding not to click on the link.
So, just saying use `OSTree static deltas` does not clarify why.

File: source/reference-manual/security/offline-keys.rst [hunk 2]
```diff
@@ -17,30 +17,171 @@ and authorize this operation using their FoundriesFactory credentials.
 This builtin `two-factor` authorization of TUF keys management operations
 makes it hard to perform known attacks by obtaining only one of the secrets.
 
-FoundriesFactory provides the ability to manage TUF keys via dedicated `Fioctl™ <Fioctl_>`_ commands.
+FoundriesFactory® provides the ability to manage TUF keys via dedicated `Fioctl™ <Fioctl_>`_ commands.
+
+TUF Keys And Their Importance
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst:22]: If you aren't familiar with TUF or its integration with our software, navigating this content can be quite daunting. Imagine approaching this from the perspective of someone who hasn't encountered the Foundries.io solution and lacks knowledge about TUF. The content presented here becomes a challenging read without some foundational understanding.

I believe we need to structure our documentation with the assumption that the reader may not have expertise in either area. After all, someone well-versed in both topics wouldn't be seeking clarity from this guide. For me personally, it took multiple readings to grasp the concepts, indicating that we might need to simplify or provide more introductory context.

File: source/reference-manual/ota/production-targets.rst [hunk 3]
```diff
@@ -45,9 +51,24 @@ To start the production release process, a user would create a new wave using th
...
+
+   .. code-block:: shell
+
+       409 CONFLICT
+       = Conflict: /ota/repo/qemu/api/v1/user_repo/root/updates?
+       = Only one TUF root updates transaction can be active at a time
+
+   It indicates a conflict with an active TUF root update.
+   The Factory admin might able to sort it out by running ``fioctl keys tuf updates cancel``.
+   After that, it is possible generate a new ``targets.key.tgz``.
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/ota/production-targets.rst:71]: @kprosise @angolini have you any idea where/how to provide the info?

File: source/reference-manual/ota/production-targets.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (vkhoroz) [reply source/reference-manual/ota/production-targets.rst:57]: There are two perspectives here:
- if a user cares what we recommend, or plans to use static deltas anyway - they would go and read.
- if a user does not care - there is no text that would motivate them to follow that link.

So, we made a recommedation; we also remind about it when the user creates a wave.
There is nothing more we can do, and nothing more we should do.

Conciseness is a tool that must be mastered and used in everyday life.
I feel like I always create _too wordy docs_; and you propose to make them even _wordier_.

Just think about that.

File: source/reference-manual/security/offline-keys.rst [hunk 4]
```diff
@@ -17,30 +17,171 @@ and authorize this operation using their FoundriesFactory credentials.
 This builtin `two-factor` authorization of TUF keys management operations
 makes it hard to perform known attacks by obtaining only one of the secrets.
 
-FoundriesFactory provides the ability to manage TUF keys via dedicated `Fioctl™ <Fioctl_>`_ commands.
+FoundriesFactory® provides the ability to manage TUF keys via dedicated `Fioctl™ <Fioctl_>`_ commands.
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst:20]: It seems marked, see ``Fioctl™ <Fioctl_>`_` and it is as it was before so I am not sure why it raises the error.
Maybe @kprosise can give me a hand here to know what to do ?
Thank you

[the remark this answers is not in the document]
Comment (vkhoroz) [reply source/reference-manual/security/offline-keys.rst:20]: The FoundriesFactory must not be marked here.

Please, follow the style guide.
It would be the best if you read it in full, sleep a few days with that in mind, and then read it again.
Only after that point you could start adding some documentation content.

It turn out to be as hard as coding; just an advice based on my personal experience.

**Comment by mike-sul** [comment]
IMO, I would let customer support team representatives to judge about the given doc/TUF clarification. Personally, as for me, it's really hard to assess objectively this document explanation in terms of helping customers to grok the TUF stuff. The key reason, I've working with TUF many years and know our TUF related implementation from inside.

@caiotpereira @vanmaegima Please, look into this doc and provide your feedback.

File: source/reference-manual/ota/production-targets.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/ota/production-targets.rst:57]: Certainly! Here's a revised version of what you wrote:

In my humble opinion (IMHO), it's not just about what we recommend. If you tell me, "I recommend 'A' because it will benefit you in 'Y' way," then I might take it seriously. Otherwise, the information might just be overlooked.

I believe it's much more appealing when we highlight the end goal or the "why" behind our recommendations rather than relying solely on technical jargon because it has so many and is hard to have the appetize to click on every note to try to understand each term to see if I want one day or not check that.

On top of that, if I am following the doc for the first time (we link it in the portal) I will know that I am doing the weave but have something that I should look after because will optimize my update.

File: source/reference-manual/security/offline-keys.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst:20]: Just to clarify:

I added the maker in `FoundriesFactory ` to see if the error disappear.
The error does not seems to be about `FoundriesFactory `

> `'Fioctl' should be marked as a trademark first time it occurs in body of text`

But I will look on that thx for your sharing.

[the remark this answers is not in the document]
Comment (vkhoroz) [reply source/reference-manual/security/offline-keys.rst:20]: Good, now one of 2 errors disappeared and I see only one of them.
Which makes me recollect that our `fio-styles` needs to be improved to understand the word boundaries.

@kprosise do you think it is worth spending time to enhance the vale rule on this?

@camilamacedo86 feel free to ignore this exact linter error.
The docs linter is not yet ideal; occasionally it produces false-positives.

File: source/reference-manual/ota/production-targets.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (vkhoroz) [reply source/reference-manual/ota/production-targets.rst:57]: Maybe this can be a good compromise:
```suggestion
We recommend that a user generates :ref:`OSTree static deltas<ref-static-deltas>` to optimize the OTA update download, before rolling out waves to devices.
```
Although (as a user) I would hardly understand what the _optimize the OTA update download_ means.
But, maybe, that misunderstanding would add `1%` chance to their desire to follow the link, so these 6 extra words are not wasted.

Just let @kprosise check the grammar.

File: source/reference-manual/security/offline-keys.rst [hunk 5]
```diff
@@ -179,18 +348,37 @@ See the section `How to View Offline TUF Keys`_ to understand the keys tarball i
...
+    - **Set Signature Threshold**: Define the necessary number of signatures for TUF root or production targets using ``fioctl keys tuf updates set-threshold``.
+    - **Offline Signing**: Sign the staged TUF root using ``fioctl keys tuf updates sign`` and sign production targets with ``fioctl keys tuf updates sign-prod-targets``.
+
+For a comprehensive list of commands and their usage examples, refer to the ``fioctl keys tuf updates --help``
+command output or consult the documentation.
+
+.. note::
+    We recommend running the command ``fioctl keys tuf updates review`` from time to time.
+    It will show you a list of our recommendations about improving your TUF usage, which evolve over time.
+    For example, it can show you the up to date information about the `Recommended Offline TUF Keys Schema`_.
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst:240]: The note was already present; I've simply relocated it to a section where it's more contextually relevant.
See:https://github.com/foundriesio/docs/pull/610/files#diff-1a60cb77c552e034077f6d892089f5994783f2a828c2e2d35468e69163fa47dcL22-L25

File: source/reference-manual/ota/production-targets.rst [hunk 6]
```diff
@@ -35,6 +35,12 @@ Waves allow Factory operators to control an exact time when devices see a new ve
 Performing a Production OTA
 ---------------------------
 
+.. important::
+
+   Before performing a Production OTA, ensure you have the `targets.key.tgz` in your
+   possession. This key is crucial for the operation and should be provided by the Factory admin.
+   For a deeper understanding of how this key is generated, refer to :ref:`ref-offline-keys`.
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/ota/production-targets.rst]: It is addressed so I am closing this one.
However, if you see fit please feel free to re-open

File: source/reference-manual/ota/production-targets.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/ota/production-targets.rst]: It is addressed so I am closing this one.
However, if you see fit please feel free to re-open

File: source/reference-manual/security/offline-keys.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst]: It is addressed so I am closing this one.
However, if you see fit please feel free to re-open

File: source/reference-manual/security/offline-keys.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst]: It is addressed so I am closing this one.
However, if you see fit please feel free to re-open

File: source/reference-manual/ota/production-targets.rst [hunk 7]
```diff
@@ -45,9 +51,24 @@ To start the production release process, a user would create a new wave using th
 This creates a new TUF targets role version for production devices which listen to OTA updates for the ``production`` tag.
 That TUF targets role only includes a single Target from CI build (in above example, that target version is 42).
 
-.. note::
+.. important::
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/ota/production-targets.rst]: It is addressed so I am closing this one.
However, if you see fit please feel free to re-open

**Comment by camilamacedo86** [comment]
HI @mike-sul @vkhoroz 

Thank you for all your feedback and inputs
It was a draft and it was not ready to review yet.
Anyway, that was good and I mainly keep what is essential. 

Again thx for all.

File: source/reference-manual/security/offline-keys.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (vkhoroz) [reply source/reference-manual/security/offline-keys.rst:197]: My comment was about a few dozen of lines, not a single note ;-)
Now it looks much better.

But, please, move it to the original location.
That was an ask of the documentation team to put this note on the top to increase its visibility.
I don't remember the PR number with that discussion already.

File: source/reference-manual/security/offline-keys.rst [hunk 8]
```diff
@@ -40,7 +35,7 @@ Thereafter, that (offline) TUF root key must be rotated regularly (at the least
   Factory owners who received it and have not yet downloaded/rotated the initial keys,
   can proceed with the initial rotation as described in this section.
 
-To take your TUF root key offline by performing its first time rotation, use this Fioctl_ command::
+To take your TUF root key offline by performing its first time rotation, use this `Fioctl™ <Fioctl_>`_ command::
```
Comment (vkhoroz) [inline source/reference-manual/security/offline-keys.rst:38]: The TM should appear only at the first mention, which is at line 20.
Forget that vale warning; style guide has a higher priority than some helper tool.

File: source/reference-manual/ota/production-targets.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (angolini) [reply source/reference-manual/ota/production-targets.rst:68]: I can understand the point. And I think I agree.
I think he is pointing here https://docs.foundries.io/latest/user-guide/troubleshooting/troubleshooting.html

@vkhoroz, do you think a link in this current page is needed or only the new entre for FAQ?

[the remark this answers is not in the document]
Comment (angolini) [reply source/reference-manual/ota/production-targets.rst:68]: Ideally, the FAQ could be an `.inc` file we gather under a FAQ folder and include in all needed places as reference. But we are not there yet, and I don't think we have already a clear goal for the FAQ pages.

File: source/reference-manual/security/offline-keys.rst [hunk 9]
```diff
@@ -60,11 +55,14 @@ Onwards, use a shorter command to rotate your (offline) TUF root key::
...
-That will lead to the inability to deliver new :ref:`Over-the-Air (OTA) updates <ref-ota>` to your Factory devices.
-Therefore, after each TUF root key rotation, we recommend that you `Backup Offline TUF Keys`_ as described below.
+.. important::
+    You must create a backup of ``root.keys.tgz``
+
+        When rotating the TUF root key, the newly generated key is added to the keys tarball (``root.keys.tgz`` in examples).
+        That file **must never be lost**.
+        Otherwise, it will be impossible to make any future updates to the Factory TUF keys.
+        That will lead to the inability to deliver new :ref:`Over-the-Air (OTA) updates <ref-ota>` to your Factory devices.
+        Therefore, after each TUF root key rotation, we recommend that you `Backup Offline TUF Keys`_ as described below.
```
Comment (angolini) [inline source/reference-manual/security/offline-keys.rst:65]: image

I don't quite understand why it's indented. Do you mean to make line 59 the admonition title? I don't think our template can handle this. But maybe you could use a `:` at the end of the sentence. Or make it bold?

I also think the paragraph ends up being too dense, and I would break all the lines.

And "as described below" is too vague to me, I caught myself missing the how-to inside the admonition block. I would add the cross-reference instead.

**Review by angolini — APPROVED** [review]
Only few minor suggestions, please ignore then if you think they don't fit!

File: source/reference-manual/security/offline-keys.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst:197]: If you want to put in the top is OK
However, I can share with you that when I read this doc and saw the note it made no sense to me until I read everything and brought confusion.

File: source/reference-manual/ota/production-targets.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply source/reference-manual/ota/production-targets.rst:68]: Great! I believe sticking to the note approach is best, as it's the method we've consistently used in various settings, don't you think?

File: source/reference-manual/security/offline-keys.rst (hunk 8 above)
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst:38]: It is failing in the checks.
But OK I will revert that

File: source/reference-manual/security/offline-keys.rst (hunk 9 above)
Comment (camilamacedo86) [reply source/reference-manual/security/offline-keys.rst:65]: The idea was just to keep in important since it is important.
I will remove the intent

File: source/reference-manual/ota/production-targets.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (vkhoroz) [reply source/reference-manual/ota/production-targets.rst:68]: @angolini if you find this content to be a valuable FAQ - then we may add it to that list (thanks for the link).

As for this page, that would be too distracting IMHO.
There can happen 100500 different errors; we should not pollute every reference manual page with all of them.

File: source/reference-manual/security/offline-keys.rst (hunk 9 above)
Comment (vkhoroz) [reply source/reference-manual/security/offline-keys.rst:65]: I'm not sure I understand the intent of this admonition TBH.
Is it to scare the user away from taking their keys offline?

IMHO the documentation should not shout at the user at every corner.
Especially, for a product like ours, where complexity is inherent to its nature.

Comment (vkhoroz) [reply source/reference-manual/security/offline-keys.rst:65]: IIUC until now our common layout was to write a plain natural language in all docs, and only put an off-topic into a note.

Now this new use of admonitions/exclamations goes apart the general look-and-feel.
It definitely makes sense in certain situations.
It just makes our docs even less uniform than they currently are; all at a time we are spending so much effort to bring them to a common look-and-feel.

File: source/reference-manual/ota/production-targets.rst [hunk 10]
```diff
@@ -35,6 +35,9 @@ Waves allow Factory operators to control an exact time when devices see a new ve
 Performing a Production OTA
 ---------------------------
 
+.. note::
+    The process for performing a Production OTA needs the :ref:`ref-offline-keys` (``targets.key.tgz``) which will be used to sign production targets.
+
 A user should define a process to select CI builds which need to be delivered to production devices.
 Let's assume a user selected a CI build version 42 as ready to be run in production.
 To start the production release process, a user would create a new wave using the below command::
```
Comment (vkhoroz) [inline source/reference-manual/ota/production-targets.rst:43]:
```suggestion
A customer should define a process to select CI builds which need to be delivered to production devices.
Every user performing production OTAs should generate their personal :ref:`offline TUF targets key <ref-offline-keys>` used to sign production targets.
Let's assume a user selected a CI build version 42 as ready to be run in production.
To start the production release process, a user would create a new wave using the below command::
```

An idea is this:
- in the past the text only told that a user needs to define the release process.
- now, you made a valid point about the prerequisite for a user to generate the targets key.
- we need to organically combine these two sentences into the whole.

In essence:
- the process requirement goes from the customer as an organization; so I changed that sentence.
- the key requirement goes to _every_ user performing production OTAs.
- furthermore, it is important to not only generate that key, but also that every such user has their _personal_ key.

File: source/reference-manual/security/offline-keys.rst [hunk 11]
```diff
@@ -108,9 +109,10 @@ This will perform the following steps:
   That approach makes it possible to distribute the targets key among a wider set of Factory admins,
   and allow them to sign production targets without exposing the TUF root key to the wider audience.
 
-After each TUF targets key rotation we recommend that you `Backup Offline TUF Keys`_ as described below.
-If you lose the offline TUF targets key, a new key can be generated if you have your Factory offline TUF root key.
-However, losing this key may be inconvenient if more than one Factory admin can manage production targets.
+.. important::
+  After each TUF targets key rotation we recommend that you `Backup Offline TUF Keys`_ as described below.
+  If you lose the offline TUF targets key, a new key can be generated if you have your Factory offline TUF root key.
+  However, losing this key may be inconvenient if more than one Factory admin can manage production targets.
```
Comment (vkhoroz) [inline source/reference-manual/security/offline-keys.rst:115]: I'm fine to use an admonition here and above, if @angolini and @kprosise agree on that.

_TL;DR_
Just IMHO, it is more relevant for the TUF root key, and less relevant for the TUF targets key.
So, I'd only leave the first one, and keep the other as a plain paragraph.
Otherwise, the user might be under false impression that both things are equally important.
Counter-intuitively, the more warnings we put the more likely people are to ignore them.

File: source/reference-manual/security/offline-keys.rst [hunk 12]
```diff
@@ -60,11 +60,12 @@ Onwards, use a shorter command to rotate your (offline) TUF root key::
...
-That file **must never be lost**.
-Otherwise, it will be impossible to make any future updates to the Factory TUF keys.
-That will lead to the inability to deliver new :ref:`Over-the-Air (OTA) updates <ref-ota>` to your Factory devices.
-Therefore, after each TUF root key rotation, we recommend that you `Backup Offline TUF Keys`_ as described below.
+.. important::
+   When rotating the TUF root key, the newly generated key is added to the keys tarball (``root.keys.tgz`` in examples).
+   That file **must never be lost**.
+   Otherwise, it will be impossible to make any future updates to the Factory TUF keys.
+   That will lead to the inability to deliver new :ref:`Over-the-Air (OTA) updates <ref-ota>` to your Factory devices.
+   Therefore, after each TUF root key rotation, we recommend that you `Backup Offline TUF Keys`_ as described below.
```
Comment (camilamacedo86) [inline source/reference-manual/security/offline-keys.rst:68]: Hi @vkhoroz,

You made a comment under another one and it was confuse so, just replying here

> 
> [vkhoroz](https://github.com/vkhoroz) [2 hours ago](https://github.com/foundriesio/docs/pull/610#discussion_r1346205014)
> I'm not sure I understand the intent of this admonition TBH.
> Is it to scare the user away from taking their keys offline?
> 
> IMHO the documentation should not shout at the user at every corner.
> Especially, for a product like ours, where complexity is inherent to its nature.
> 
> Member
> @[vkhoroz](https://github.com/vkhoroz) vkhoroz [2 hours ago](https://github.com/foundriesio/docs/pull/610#discussion_r1346210528)
> IIUC until now our common layout was to write a plain natural language in all docs, and only put an off-topic into a note.
> 
> Now this new use of admonitions/exclamations goes apart the general look-and-feel.
> It definitely makes sense in certain situations.
> It just makes our docs even less uniform than they currently are; all at a time we are spending so much effort to bring them to a common look-and-feel. 

**Intention/change:** 
- Same text/Same default DOC layout just add it into an important note
- WHY? it lets users know that they cannot lose the key Otherwise they cannot update their devices in prod.

Therefore it seems important to me.

So, @vkhoroz

Are you saying that we should not use the default doc layout here to highlight this info?  
Do you want to revert it?

Comment (vkhoroz) [reply source/reference-manual/security/offline-keys.rst:68]: @camilamacedo86 You're making a reply to an old comment to an old version of the change.
But, these lines were already changed _after_ that old discussion.
In fact, your new changes addressed that comment to certain extent.

So, it is better to close this than continue the discussion.
We now have a new code, and a new (relevant) discussion about it: https://github.com/foundriesio/docs/pull/610#discussion_r1346263012.

**Review by vkhoroz — APPROVED** [review]
I'm fine with the current version.

@kprosise needs to approve.

**Review by camilamacedo86 — COMMENTED** [review]
In this PR, I've mainly let the text suggested by @vkhoroz , which seems to be appropriate.  I believe this is the best approach. We can discuss other changes or additions in follow-up PRs.Sorry for any inconvenience or disturbance I may have caused.

Thank you for your attention and time
I hope it small nit changes are good to fly,
Please, let me know if any further changes would be required.
